src/sales/order-status.md

```diff
@@ -11,18 +11,18 @@ _Order Status_
 
 ## Predefined Order Status
 
-|Order Status|Status Code|
-|--- |--- |
-|Processing|processing|
-|Suspected Fraud|fraud|
-|Pending Payment|pending_payment|
-|Payment Review|payment_review|
-|Pending|pending|
-|On Hold|holded|
+|Order Status|Status Code||
+|--- |--- |--- |
+|Processing|processing|When the state of new orders is set to “Processing”, the _Automatically Invoice All Items_ option becomes available in the configuration.|
```

Comment:
We are avoiding the use of quotes for literal values.

```suggestion
|Processing|processing|When the state of new orders is set to `Processing`, the _Automatically Invoice All Items_ option becomes available in the configuration.|
```

Reply:
Done!